packages/app-mobile/components/global-style.ts

```diff
@@ -1,6 +1,18 @@
-const Setting = require('@joplin/lib/models/Setting').default;
+import Setting from '@joplin/lib/models/Setting';
 const { Platform } = require('react-native');
-const { themeById } = require('@joplin/lib/theme');
+import { themeById } from '@joplin/lib/theme';
+
+export interface Style {
+	[key: string]: any;
+}
+
+interface Fonts {
+	[key: number]: string;
+}
+
+interface ThemeCache {
+	[key: string]: any;
```

Comment:
Shouldn't that be "Theme" instead of "any"? (Not sure it can work but please try)

Reply:
It threw an error: "Type 'Style' is missing the following properties from type 'Theme': appearance, backgroundColor, backgroundColorTransparent, oddBackgroundColor, and 32 more"

Reply:
And a couple of other errors in other files.

Reply:
Ok let's keep it "any" then